gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -222,24 +215,73 @@ def _add_format_fields(schema, formats):
   for field in schema.fields:
     if field.name in _CONSTANT_CALL_FIELDS:
       continue
-    formats.update({
-        field.name: _Format(field.name,
+    if field.name in vcf_reserved_fields.FORMAT_FIELDS.keys():
+      _add_reserved_field(field,
+                          vcf_reserved_fields.FORMAT_FIELDS.get(field.name),
+                          formats)
+    else:
+      formats.update({
+          field.name: _Format(field.name,
+                              parser.field_counts[vcfio.MISSING_FIELD_VALUE],
+                              _BIG_QUERY_TYPE_TO_VCF_TYPE_MAP.get(field.type),
+                              field.description)})
+
+
+def _add_info_fields(field, infos):
+  # type: (bigquery.TableFieldSchema, Dict[str, _Info]) -> None
+  if field.name == bigquery_util.ColumnKeyConstants.ALTERNATE_BASES:
+    _add_info_fields_from_alternate_bases(field, infos)
+  else:
+    if field.name in vcf_reserved_fields.INFO_FIELDS.keys():
+      _add_reserved_field(field,
+                          vcf_reserved_fields.INFO_FIELDS.get(field.name),
+                          infos)
+    else:
+      infos.update({
+          field.name: _Info(field.name,
                             parser.field_counts[vcfio.MISSING_FIELD_VALUE],
                             _BIG_QUERY_TYPE_TO_VCF_TYPE_MAP.get(field.type),
-                            field.description)})
+                            field.description,
+                            None,
+                            None)})
 
 
 def _add_info_fields_from_alternate_bases(schema, infos):
   # type: (bigquery.TableFieldSchema, Dict[str, _Info]) -> None
   for field in schema.fields:
     if field.name in _CONSTANT_ALTERNATE_BASES_FIELDS:
       continue
+    if field.name in vcf_reserved_fields.INFO_FIELDS.keys():
+      _add_reserved_field(field,
```

Comment:
In general, it's better to limit the scope and 'passing around' of mutable objects so that it's more clear/readable how they're bing mutated. So, I think we should have a `_validate_reserved_field` function instead that gets called by default (unless we add an explicit `allow_malformed_*` flag). If it passes, then you can get the reserved field like you've done now and call `infos.update(..)` on it. This also allows us to use the description from the BQ schema (if provided) and don't always overwrite it as people may add more things to the description.